stable/sonarqube/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: sonarqube
 description: Sonarqube is an open sourced code quality scanning tool
-version: 0.13.5
-appVersion: 7.6
+version: 0.14.0
+appVersion: 7.4
```

Comment:
Please update the appVersion to 7.6 to reflect the recent change

Reply:
I fixed the wrong rebase. 
The appVersion should now be unchanged by this PR.